### PR TITLE
Main game component

### PR DESCRIPTION
**Pull request by y-golde**

### CODE REVIEW

File: Client/src/Components/App/Game/GamePanel/GamePanel.svelte [hunk 1]
```diff
@@ -1,9 +1,44 @@
...
+	<div class="keyboard-container mx-auto">
+		<Keyboard bind:disabledChars />
+	</div>
+</div>
 
 <style>
 	.panel-card {
 		/* stub */
-		min-height: 400px;
+		min-height: 700px;
```
Comment (dstr951) [inline Client/src/Components/App/Game/GamePanel/GamePanel.svelte:41]: why minHeight? doesn't the keyboard take enough space?

File: Client/src/Components/App/Game/GamePanel/Keyboard/Keyboard.svelte [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+	export let disabledChars: string[];
+
+	let { isDisabled } = UseKeyboard(disabledChars);
+
+	$: {
+		isDisabled = UseKeyboard(disabledChars).isDisabled;
+	}
+</script>
+
+<div class="flex flex-wrap gap-4 text-2xl justify-center keyboard">
```
Comment (dstr951) [inline Client/src/Components/App/Game/GamePanel/Keyboard/Keyboard.svelte:15]: maybe the keyboard should be a little smaller?
It takes almost half of my laptop screen
image

**Review by dstr951 — APPROVED** [review]
basically approved

File: Client/src/Components/App/Game/GamePanel/GamePanel.svelte (hunk 1 above)
Comment (y-golde) [reply Client/src/Components/App/Game/GamePanel/GamePanel.svelte:41]: its a stub - for later

File: Client/src/Components/App/Game/GamePanel/Keyboard/Keyboard.svelte (hunk 2 above)
Comment (y-golde) [reply Client/src/Components/App/Game/GamePanel/Keyboard/Keyboard.svelte:15]: isn't the keyboard like the main component? dont forget we need to add badges and stuff

Comment (y-golde) [reply Client/src/Components/App/Game/GamePanel/Keyboard/Keyboard.svelte:15]: I'll address this change and the other one in a seperate PR if nececcary

**Pull request merged**